Makefile

```diff
@@ -48,3 +49,9 @@ setup:
 .PHONY: test
 test: dist/ramda.js
 	npm test
+	find src -name '*.js' -not -path 'src/internal/*' \
+	| sed 's:src/\(.*\)[.]js:exports.\1 = require("./src/\1");:' >index.js
+	sed -i '' '/"main":/d' package.json
```

Comment:
Then, remove the `"main"` entry from __package.json__ so the entry point for the package is __index.js__ rather than __dist/ramda.js__.